app/javascript/components/ErrorBoundary/component.js

```diff
@@ -11,29 +18,13 @@ class ErrorBoundary extends Component {
     message: I18n.t('common.jsError'),
   };
 
-  state = {
-    hasError: false,
-  };
-
-  componentDidCatch(error, info) {
-    if (process.env.NODE_ENV !== 'test') {
-      console.error('[Boundary error]', error);
-      console.error('[Boundary info]', info);
-    }
-
-    this.setState({ hasError: true });
-  }
```

Comment:
This method is no longer explicitly needed because it's provided by Sentry.ErrorBoundary.